test/modules/standard/Random/deitz/test1D2D.chpl

```diff
@@ -113,8 +127,10 @@ config const n: int = 8;
   var A: [{1..n} dmapped Block(rank=1,boundingBox={1..n/2})] real;
   forall (r,a) in zip(rs.iterate(A.domain), A) do
     a = r;
-  writeln(for e in A do format("#######", e.locale.id));
-  writeln(for e in A do format("#.#####", e));
+  for e in A do writef(" #######", e.locale.id);
+  writeln();
+  for e in A do writef(" #.#####", e);
+  writeln();
```

Comment:
I'm OK either way on these but I have a slight preference to revert the tests that used format in this way and
also leave the testformat tests there. I'll modify the testformat tests to work with whatever happens with format().